test/systemtests/docker_test.go

```diff
@@ -603,6 +603,38 @@ func (d *docker) checkSchedulerNetworkCreated(nwName string, expectedOp bool) er
 	return err
 }
 
+func (d *docker) checkSchedulerNetworkOnNodeCreated(nwNames []string, n *node) error {
+	ch := make(chan error, 1)
+	for _, nwName := range nwNames {
+		go func(nwName string, n *node, ch chan error) {
+			logrus.Infof("Checking whether docker network %s is created on node %s", nwName, n.Name())
+			cmd := fmt.Sprintf("docker network ls | grep netplugin | grep %s | awk \"{print \\$2}\"", nwName)
```

Comment:
Should we just do etcdctl ls /contiv.io? That way, it will work for k8s as well...